app/views/user/account/edit.slim

```diff
@@ -41,7 +52,13 @@ main
         = f.label :primary_circle_id
         = f.collection_select :primary_circle_id, @form.primary_circle_options, :id, :name
         = errors.message_for(:primary_circle_id)
-
+        
+    .field-row
+      = f.check_box :public_profile
+      = " "+User.human_attribute_name("public_profile")
```

Comment:
Don't use a Ruby `String` to add padding. Ruby is for functionality, CSS is for styling.